src/Mcfedr/AwsPushBundle/Resources/config/services.yml

```diff
@@ -8,12 +8,14 @@ parameters:
 services:
     mcfedr_aws_push.devices:
         class: Mcfedr\AwsPushBundle\Service\Devices
+        public: true
         arguments:
           - "@mcfedr_aws_push.sns_client"
           - "%mcfedr_aws_push.platforms%"
 
     mcfedr_aws_push.messages:
```

Comment:
Can we change the service name to be the class, and add alias for backwards compatibility